app/move/middleware.js

```diff
@@ -14,6 +14,8 @@ module.exports = {
 
     try {
       req.move = await req.services.move.getById(moveId)
+      req.journeys = await req.services.journey.getByMoveId(moveId)
```

Comment:
I'm wondering if this belongs in its own middleware, `setJourneys` for example, as then it can be tested independently.

We could also then refactor the code a little bit so [the journeys controller](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/main/app/move/controllers/journeys.js) uses the same middleware.